src/ansys/fluent/core/launcher/launcher.py

```diff
@@ -405,6 +405,15 @@ def scm_to_py(topy):
     return launch_string
 
 
+class LaunchFluentError(Exception):
+
+    def __init__(self, launch_string):
+        identifier = "fluent.exe"
```

Comment:
This won't work for Linux.

I feel it is sufficient to show just the whole `launch_string`, no need to break inside it

Reply:
Good point @mkundu1

Reply:
Thank you @mkundu1. This has been updated.